src/help/components/entry/EntryPreview.tsx

```diff
@@ -0,0 +1,183 @@
+          size: '0',
+          queryFacets: [`category:${category}`],
+          facets: null,
+        }),
+      [category]
+    )
+  );
+
+  const total = +(headers?.['x-total-records'] || 0);
+
+  if (!headers) {
+    return <Chip>{category}</Chip>;
+  }
+
+  return (
+    <Chip>
+      {category} ({total} {pluralise('article', total)}{' '}
+      {total === 0 ? '❌' : '✅'})
+    </Chip>
+  );
+};
+
+const EntryPreview = (props: RouteChildrenProps<{ accession: string }>) => {
+  const [url, setUrl] = useState('');
+  const { data } = useDataApi<string>(url);
+
+  const [parsedData, otherAttributes] = useMemo<
+    [
+      data: HelpEntryResponse,
+      otherAttributes?: Record<string, string | undefined>
+    ]
+  >(() => {
+    if (!data) {
+      return [defaultData];
+    }
+    const {
+      attributes: { title, categories, ...otherAttributes },
+      body,
+    } = fm<{
+      title?: string;
+      categories?: string;
+      [key: string]: string | undefined;
+    }>(data.trim());
+    const content = cleanText(marked(body), getCleanTextOptions('h1'));
+    return [
+      {
+        id: '_preview',
+        title: title || '',
+        categories:
+          categories?.replaceAll('_', ' ')?.split(arraySeparator) || [],
+        content,
+        lastModified: new Date().toISOString(),
+      },
+      otherAttributes,
+    ];
+  }, [data]);
+
+  const handleChange = useCallback((event: ChangeEvent<HTMLInputElement>) => {
+    const url = event.target.value.trim();
+    if (!url || !url.startsWith('https://')) {
+      return;
+    }
+    if (url.startsWith('https://github.com/')) {
+      // we don't have the raw file, try to find it
+      const ghWikiMatch = url.match(ghWikiRE);
+      if (ghWikiMatch?.groups) {
+        setUrl(
+          `https://raw.githubusercontent.com/wiki/${ghWikiMatch.groups.user}/${ghWikiMatch.groups.project}/${ghWikiMatch.groups.file}.md`
+        );
+        return;
+      }
+
+      const ghNormalMatch = url.match(ghNormalRE);
+      if (ghNormalMatch?.groups) {
+        setUrl(
+          `https://raw.githubusercontent.com/${ghNormalMatch.groups.user}/${ghNormalMatch.groups.project}/${ghNormalMatch.groups.filepath}`
+        );
+        return;
+      }
+    }
+    setUrl(url);
+  }, []);
+
+  // Note: copy structure of "normal" help entry page in order to preview it
+  // Maybe we should just use the actual help entry page?
+  return (
+    <>
+      <HTMLHead>
+        {/* Don't index 4xx pages */}
+        <meta name="robots" content="noindex" />
+      </HTMLHead>
+      <section className={styles['private-area']}>
+        <div>The content outside of this area is a preview</div>
+        <label>
+          Load article content from a URL:{' '}
+          <input type="url" onChange={handleChange} />
+        </label>
+        {data && (
+          <output>
+            <details>
+              <summary>Raw loaded content</summary>
+              <CodeBlock>{data}</CodeBlock>
```

Comment:
As some of these files can be really long what do you think about doing something like:
```
max-height: 40vh;
overflow: auto;
```

Reply:
Changed in https://github.com/ebi-uniprot/uniprot-website/pull/539/commits/77e306f69c6cc29745cbbbf70e826b1e14908ce8